src/js/index.js

```diff
@@ -83,21 +83,22 @@ function createMemo(id, text, position, size) {
   close.addEventListener("touchend", handleMemoClose);
   memo.appendChild(close);
 
-  /** toggle between markdown and text **/
-  const mdToggle = document.createElement("div");
-  mdToggle.classList.add("mdToggle");
-  mdToggle.innerText = "preview";
-  mdToggle.addEventListener("mouseup", function () {
+  /** toggle between edit and preview mode **/
+  const modeToggle = document.createElement("div");
+  modeToggle.classList.add("modeToggle");
+  modeToggle.innerText = "preview";
+  var toggleFn = function () {
```

Comment:
1. Is there a reason why this was function was made as a variable instead of a const?
2. This doesn't really follow the current pattern of keeping multi-use functions outside of the parent function scope.

Reply:
1. `var` at line 90 can be made a `const`.
2. The function `toggleFn` currently doesn't have any purpose outside the `createMemo` function that why i kept it inside. But if required can be extracted into global function.

Let me know if need to update the code.